.github/workflows/bundle-size.yml

```diff
@@ -8,10 +8,9 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v2-beta
+    - uses: actions/checkout@v2
       with:
         fetch-depth: 1
```

Comment:
This is now [the default](https://github.com/actions/checkout/blob/aabbfeb2ce60b5bd82389903509092c4648a9713/action.yml#L56-L58).

```suggestion
```